File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/19.go
package main
import "fmt"
import "strings"

func contem010(s string) bool {
    return strings.Contains(s, "010")
}

func main() {
    fmt.Println(contem010("001010"))  // True
    fmt.Println(contem010("1101"))    // False
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/19.java
public class AFNContem010 {
    public static boolean contem010(String s) {
        return s.contains("010");
    }

    public static void main(String[] args) {
        System.out.println(contem010("001010"));  // True
        System.out.println(contem010("1101"));    // False
    }
}
